saleor/settings.py

```diff
@@ -949,3 +951,29 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
 # Transaction items limit for PaymentGatewayInitialize / TransactionInitialize.
 # That setting limits the allowed number of transaction items for single entity.
 TRANSACTION_ITEMS_LIMIT = 100
+
+
+def set_up_garbage_collector_trace(phase, info):
```

Comment:
I don't think this belongs in `settings`

Reply:
Yeah, I wasn't sure, that settings is the proper place for that, but didn't think about core app. Thanks! Will change this.